src/plugin_nng/plugin_nng.h

```diff
@@ -0,0 +1,2 @@
+void StartPluginNng();
```

Comment:
Probably can include a stub.h file for these and include them when the nng plugin flag is off in cmake. It'll be nice and clean that way.